src/constants.ts

```diff
@@ -89,6 +89,14 @@ export const FONT_FAMILY = {
   Cascadia: 3,
 };
 
+export const WINDOWS_EMOJI_FALLBACK_FONT = "'Segoe UI Emoji'";
+
+export const FONT_FAMILY_FALLBACKS = {
+  [FONT_FAMILY.Virgil]: `'Comic Sans MS', 'Segoe Print', 'Bradley Hand', 'Lucida Handwriting', 'Marker Felt', cursive, ${WINDOWS_EMOJI_FALLBACK_FONT}`,
```

Comment:
Personally I find [Segoe Print](https://learn.microsoft.com/en-us/typography/font-list/segoe-print#style--weight-examples) a more faithful replacement for Virgil than [Comic Sans MS](https://learn.microsoft.com/en-us/typography/font-list/comic-sans-ms#style--weight-examples), so would suggest perhaps swapping their order.

![image](https://github.com/excalidraw/excalidraw/assets/1007415/2cb944d5-c40b-43af-b430-952f88e5c2fe)